bin/integrate_pr.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+while [[ $# > 1 ]]
+do
+key="$1"
+case $key in
+    -r|--remote)
+    REMOTE="$2"
+    shift
+    ;;
+    -pr|--pull-request)
+    PULL_REQUEST="$2"
+    shift
+    ;;
+    -db|--destination-branch)
+    DESTINATION_BRANCH="$2"
+    shift
+    ;;
+    --dry-run)
+    DRY_RUN=--dry-run
+    shift
+    ;;
+    --setup)
+    SETUP=1
+    ;;
+    *)
+    shift
+    ;;
+esac
+shift
+done
+
+if [ "x$SETUP" != "x" ] && [ "x$REMOTE" != "x" ]; then
+
+	echo "Setting up remote for Pull Request integration"
+	git config --replace-all remote.$REMOTE.fetch "+refs/heads/*:refs/remotes/$REMOTE/*"
+	git config --add remote.$REMOTE.fetch "+refs/pull/*:refs/remotes/$REMOTE/pr/*"
+	echo "Fetching new refspecs"
+	git fetch $REMOTE --prune
```

Comment:
Why change the config and not fetch a single ref like this: `git fetch upstream refs/pull/1/head:bla`?